ro.go.hmlendea.Duolingo.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
```

Comment:
I do not understand while you brought it back?

Reply:
I've had trouble with the icon (but turned out it was something else) and the Exec (but I fixed it another way)